### ISSUE TITLE
Prints text in zsh before command

### ISSUE THREAD

**Issue by soulofmischief** [yasyf/gpt-do#6]
After pressing `y` to accept a command in zsh, several errors are printed to the console:

```
» ddo echo hello                                                                                                                                                                                   3:27PM
This command will print the string 'hello' to the terminal.
echo hello
Do you want to continue? [y/N]: y
/home/user/.zsh/oh-my-zsh/oh-my-zsh.sh:46: ZSH_CACHE_DIR: parameter not set
/home/user/.zsh/config/zshrc.d/044_functions-vcsh.zsh:3: no match
/home/user/.zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh:42: unmatched "
/home/user/.zsh/plugins/hhighlighter/h.sh:1: unmatched "
/home/user/.zsh/config/zshrc.d/061_plugins-hstr.zsh:4: command not found: bindkey
compinit:170: unmatched '
/home/user/.bun/_bun:630: command not found: compdef
/home/user/.zsh/oh-my-zsh/oh-my-zsh.sh:46: ZSH_CACHE_DIR: parameter not set
/home/user/.zsh/config/zshrc.d/044_functions-vcsh.zsh:3: no match
/home/user/.zsh/plugins/zsh-autosuggestions/zsh-autosuggestions.zsh:42: unmatched "
/home/user/.zsh/plugins/hhighlighter/h.sh:1: unmatched "
compinit:170: unmatched '
/home/user/.bun/_bun:630: command not found: compdef
hello
```

**Comment by yasyf** [comment]
I'll look into this!

**Comment by yasyf** [comment]
This should be fixed in `0.1.16`! Can you confirm?

**Comment by soulofmischief** [comment]
It works! I think! Thank you!
